include/particle.tcc

```diff
@@ -213,37 +214,49 @@ bool mpm::Particle<Tdim, Tnphases>::map_mass_momentum_to_nodes(unsigned phase) {
 template <unsigned Tdim, unsigned Tnphases>
 void mpm::Particle<Tdim, Tnphases>::compute_strain(unsigned phase, double dt) {
   // Strain rate
-  Eigen::VectorXd strain_rate = cell_->compute_strain_rate(bmatrix_, phase);
-  // dstrain
-  Eigen::Matrix<double, 6, 1> dstrain;
-  dstrain.setZero();
+  Eigen::VectorXd cell_strain_rate =
```

Comment:
It's not the strain rate of the cell, which is ambiguous, rename to particle_strain_rate